tokio-trace/tokio-trace-core/src/subscriber.rs

```diff
@@ -187,13 +217,22 @@ pub trait Subscriber: 'static {
 
     /// Records that an [`Event`] has occurred.
     ///
+    /// This method will be invoked when an Event is constructed by
+    /// the `Event`'s [`dispatch` method]. This happens internally
```

Comment:
Nit: I might add a 
```suggestion
    /// the `Event`'s [`dispatch` method]. For example, this happens internally
```
here --- the `tokio-trace` macros aren't the _only_ case where `Event::dispatch` will be called.